src/osdproc.c

```diff
@@ -391,30 +462,31 @@ void draw_radar() {
 
 
   write_filled_rectangle_lm(x - 9, y + 6, 15, 9, 0, 1);
-  sprintf(tmp_str, "%d", (int)osd_pitch);
+  sprintf(tmp_str, "%d", (int)osdproc_osd_state.osd_pitch);
   write_string(tmp_str, x, y + 5, 0, 0, TEXT_VA_TOP, TEXT_HA_CENTER, 0, SIZE_TO_FONT[0]);
 
-  y = eeprom_buffer.params.Atti_mp_posY - (int)(38.0f * atti_mp_scale);
+  y = eeprom_buffer.params.Atti_mp_posY - (int)(38.0f * TEMP_atti_mp_scale);
```

Comment:
to we really need the `TEMP_` prefix? Since it seems like it is the regularly used value it seems (at first glance) to be fine without it (aka the way it used to be)